libs/crypto/src/mcl_dkg.cpp

```diff
@@ -33,6 +33,8 @@ namespace mcl {
 
 std::atomic<bool> details::MCLInitialiser::was_initialised{false};
 
+details::MCLInitialiser execute_me_before_runtime;
```

Comment:
Note: so long as this is linked, it should call the MCL init just before main is called. This might cause undefined behaviour if any MCL functions are called in other constructors so let's avoid this.